tooling/image-updater/internal/clients/acr.go

```diff
@@ -108,3 +108,10 @@ func (c *ACRClient) getAllTags(ctx context.Context, repository string) ([]Tag, e
 
 	return allTags, nil
 }
+
+// GetArchSpecificDigest for ACR - ACR images are always single-arch (amd64), so just return the regular digest
+// TODO: If ACR starts hosting multi-arch images in the future, implement manifest inspection here
```

Comment:
Do it now? I don't see the value of deferring this - at a minimum, detect the fact that the assumption is violated and error, don't silently return the wrong thing.